tests/python/unittest/test_meta_schedule_tune_tir.py

```diff
@@ -122,7 +121,7 @@ def _sch_rules():
                 M.MultiLevelTiling(
                     structure="SSSRRSRS",
                     tile_binds=["blockIdx.x", "blockIdx.y", "threadIdx.y"],
-                    # use_tensor_core=True,
+                    use_tensor_core=True,
```

Comment:
No I dont believe we are have tensor core rules ready on mainline

Reply:
Removed.